core/auth/src/test/java/software/amazon/awssdk/auth/signer/internal/SignerKeyTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.auth.signer.internal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import java.time.Instant;
+import org.junit.Test;
+
+public class SignerKeyTest {
+
+    @Test
+    public void isValidForDate_dayBefore() {
```

Comment:
Shouldn't it be isNotValid? and for the after method too

Reply:
I normally like to name the tests to include the name of the method I'm testing, which is `isValidForDate()` in this case :)

Reply:
I like to include the expected behavior to be sure I understand the test's intention.

Reply:
That's reasonable. Will update